docs/UserGuide.md

```diff
@@ -293,20 +307,33 @@ Example of usage:
 ```lang-none
 >> runninglow
 You are running low on food in these categories:
-1. MEAT quantity: 250 out of 500
-
->> setlimit meat /qty 300
-Okie dokie! The new minimum quantity for category 'MEAT' is 300
+1. VEGETABLE quantity: 0 out of 500
+2. FRUIT quantity: 0 out of 500
+3. SEAFOOD quantity: 0 out of 500
+4. EGG quantity: 0 out of 500
+5. DAIRY quantity: 0 out of 500
+6. BEVERAGE quantity: 0 out of 500
+7. COOKED_DISH quantity: 0 out of 500
+8. FROZEN quantity: 0 out of 500
+9. OTHER quantity: 0 out of 500
 
->> runninglow
-You are running low on food in these categories:
-1. MEAT quantity: 250 out of 300
+>> setlimit ready_to_eat /qty 600 
+Okie dokie! The new minimum quantity for category 'READY_TO_EAT' is 600
 
->> setlimit meat /qty 200
-Okie dokie! The new minimum quantity for category 'MEAT' is 200
+>> setlimit vegetable /qty -1
+Okie dokie! The new minimum quantity for category 'VEGETABLE' is -1
```

Comment:
Similarly here would be "Okie dokie! The new minimum quantity for category 'VEGETABLE' is 0"

Reply:
modified